sense-android-library/documentation/sensor_profiles.md

```diff
@@ -0,0 +1,472 @@
+    }
+  },
+  "required": [
+    "tag",
+    "type",
+    "text"
+  ]
+}
+~~~
+
+### time_active
+~~~json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "The time physically active in seconds",
+  "type": "number",
+}
+~~~
+
+
+### sleep
+*renamed from sleep_time*
+~~~json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "end_date": {
+      "description" : "The end time of the sleep period in epoch seconds",
+      "type": "number"
+    },
+    "metadata": {
+      "type": "object",
+      "properties": {
+        "core version": {
+          "description" : "The version of the cortex core",
+          "type": "string"
+        },
+        "module version": {
+          "description" : "The version of the sleep_time module",
+          "type": "string"
+        },
+        "status": {
+          "description" : "The current status of the sleep_time module for debugging module, e.g. awake: too much noise",
+          "type": "string"
+        }
+      }
+    },
+    "hours": {
+      "description" : "The number of actual sleep hours",
+      "type": "integer"
+    },
+    "start_date": {
+      "description" : "The start time of the sleep period in epoch seconds",
+      "type": "integer"
+    }
+  },
+  "required": [
+    "end_time",
+    "hours",
+    "start_time"
+  ]
+}
+~~~
+*changed the sleepTime key to hours, end_date and start_date to end_time and start_time resp.*
+
+### sleep_estimate
+*renamed from sleep_time_estimate*
+~~~json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "end_time": {
+      "description" : "The end time of the sleep period in epoch seconds",
+      "type": "number"
+    },
+    "history_based": {
+      "description" : "Whether the sleep value is based on the history or on the actual sleep period",
+      "enum": [0, 1]
+    },
+    "metadata": {
+      "type": "object",
+      "properties": {
+        "core version": {
+          "description" : "The version of the cortex core",
+          "type": "string"
+        },
+        "module version": {
+          "description" : "The version of the sleep_time module",
+          "type": "string"
+        },
+        "status": {
+          "description" : "The current status of the sleep_time module for debugging module, e.g. awake: too much noise",
+          "type": "string"
+        }
+      }
+    },
+    "hours": {
+      "description" : "The number of actual sleep hours",
+      "type": "integer"
+    },
+    "start_time": {
+      "description" : "The start time of the sleep period in epoch seconds",
+      "type": "integer"
+    }
+  },
+  "required": [
+    "end_time",
+    "hours",
+    "start_time"
+  ]
+}
+~~~
+*changed the sleepTime key to hours, end_date and start_date to end_time and start_time resp.*
+
+
+
+## Deprecated sensors
+Sensors that output data that will not be stored in the back-end.
```

Comment:
@tedschmidt i'm not sure why we don't want this sensors data to be stored in backend, but i think it will be safe to defined it just in case we need to store it later